vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/VividusToZephyrExporterApplication.java

```diff
@@ -37,10 +40,22 @@
 @SuppressWarnings("checkstyle:hideutilityclassconstructor")
 public class VividusToZephyrExporterApplication
 {
+    @Value("${zephyr.exporter.api-type}")
+    private static ZephyrApiType zephyrApiType;
+
     public static void main(String[] args) throws IOException, JiraConfigurationException
     {
         ApplicationContext ctx = SpringApplication.run(VividusToZephyrExporterApplication.class, args);
-        ZephyrExporter exporter = ctx.getBean(ZephyrExporter.class);
+
+        ZephyrExporter exporter;
+        if (zephyrApiType == ZephyrApiType.SQUAD)
```

Comment:
Why not have a proper bean depending on the settings https://github.com/vividus-framework/vividus/pull/2725#commitcomment-73018351